### ISSUE TITLE
Support vbox[width=...]

### ISSUE THREAD

**Issue by simoncozens** [simoncozens/sile#367]
First \vbox should put a box into the typesetter's queue. Second, this should make two columns:

      \begin[papersize=a5]{document}
      \script[src=packages/lorem]
      \hbox{\vbox[width=60pt]{Text here will wrap \lorem}}
      \hbox{\vbox[width=60pt]{Text here will wrap \lorem}}
      \par
      \script{SILE.typesetter:debugState()}
      \end{document}

**Comment by alerque** [comment]
Won't this need to introduce some alignment parameters too? Sort of like `display: inline-block` in CSS? Or `\begin{minipage]| in TeX. If I'm understanding this correctly the same layout logic would also be useful for image positioning.

**Comment by simoncozens** [comment]
Not really. The idea is that you put a vbox onto the horizontal list (which is supported but the output routine doesn't quite work).

Consider the following piece of plain TeX:

```TeX
hello
\qquad \vbox to 20pt {\hbox to 40pt{Stuff in a}\hbox to 25pt{vbox}}
\qquad \vbox to 20pt {\hbox to 40pt{Stuff in a}\hbox to 25pt{vbox}}
\qquad world.
\end
```

![screen shot 2016-09-05 at 16 11 55](https://cloud.githubusercontent.com/assets/106728/18238756/813091f6-7383-11e6-9b4a-c238b9e7c4f9.png)

(This is, essentially, how minipage is implemented under the hood.)
